Fix various spelling mistakes

bdrung opened the pull request:
Address spelling mistakes found by lintian.

hartwork commented on salt/modules/ceph.py:
```diff
@@ -118,7 +115,7 @@ def zap(target=None, **kwargs):
     """
     if target is not None:
-        log.warning("Depricated use of function, use kwargs")
+        log.warning("Deprecated use of function, use kwargs")
```
My vote for reverting changes to this file *except* this one, because they are not strictly spelling related.

bdrung replied:
I added commit 0bca4366663d98bae2dfbf02b4fa400e241383c2 (Drop Python 2 support from salt/modules/ceph.py) to address the failing test case. I can drop this commit again if you prefer having failed test cases.

s0undt3ch replied:
If a file is touched, it will get it's Py2 and six support removed automatically.
The failure to do so will make the PR fail.
So, while I agree that this no longer just a spelling fix, it's a necessary change.

hartwork reviewed (dismissed):
>  I can drop this commit again if you prefer having failed test cases.

I actually love failing tests… okay! :smiley: I guess it's a question of view if the CI should be fixed in here out elsewhere but you have a point. Any ideas how to re-run the two failed CI parts in here other than with a forced push?

bdrung committed:
Fix spelling mistake of deprecated

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of from

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix the wrong "allow to do" phrase

`allow to do` is not a valid English phrase. Either use a gerund (allow
doing) or add a patient/object (allow someone to do).

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of substitution

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of "in case"

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of suppress

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of functionality

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of function

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of addresses

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of contains

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>

bdrung committed:
Fix spelling mistake of library

Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>
